### PR TITLE
[JQA] CLC-6866 Support creation of historical term data in S3

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-6866

In hopes of streamlining our upcoming release process.

### CODE REVIEW

**Comment by raydavis** [comment]
Travis only reports the new-normal MyTasks fail.

File: lib/tasks/data_loch.rake [hunk 1]
```diff
@@ -16,11 +18,11 @@ namespace :data_loch do
         Rails.logger.info "Starting snapshots for term #{term_id}."
 
         courses_path = DataLoch::Zipper.zip_courses term_id
-        s3.upload_to_daily('courses', courses_path)
+        s3.upload('courses', courses_path, is_historical)
         FileUtils.rm courses_path
 
         enrollments_path = DataLoch::Zipper.zip_enrollments term_id
-        s3.upload_to_daily('enrollments', enrollments_path)
+        s3.upload('enrollments', courses_path, is_historical)
```
Comment (johncrossman) [inline lib/tasks/data_loch.rake]: @raydavis do we want `enrollments_path` above?

Comment (raydavis) [reply lib/tasks/data_loch.rake]: Why yes, I believe we do. Amending...

**Comment by johncrossman** [comment]
Once I confirm standard MyTasks failure, I will merge.

**Pull request merged**